src/lib/sdk/billing.ts

```diff
@@ -739,6 +739,14 @@ export class Billing {
         );
     }
 
+    async validateInvoice(organizationId: string, invoiceId: string): Promise<Invoice> {
+        const path = `/organizations/${organizationId}/invoices/${invoiceId}/validate`;
+        const uri = new URL(this.client.config.endpoint + path);
+        return await this.client.call('PATCH', uri, {
```

Comment:
Why is this a `PATCH`? Is this also updating the invoice on backend?
Should be a `GET` if we are just fetching the details of an invoice, right?

Reply:
Yes, it will check the state of payment and updates the invoice if required so it's a PATCH